### DISCUSSION TITLE
Increasing face detail without changing the character

### DISCUSSION THREAD

**Discussion by gabriel-oj** [lllyasviel/Fooocus#2670] [Q&A] [answered]
Hi guys, I've been trying to increase the face details of my character via inpaint, but I can't seem to increase details without also changing the character slighty. Is there a way to do this, while maintaining the exact same character? Thanks.

**Comment by mashb1t** [comment, marked as answer]
You can try to mix inpaint with image prompt (CPDS / PyraCanny), activatable in developer debug mode > control.
The results may still differ a bit from the original one though, because the AI needs some creative freedom to work with.
